actions/class.Runner.php

```diff
@@ -341,7 +341,7 @@ public function submitItemState()
         $serviceCallId = $this->getRequestParameter('testServiceCallId');
 
         try {
-            $serviceContext = $this->getServiceContext(false);
+            $serviceContext = $this->getServiceContext(true);
```

Comment:
You cannot do that, this will prevent the item state to be stored when the test session is suspended. The parameter is to allow the session check.

By example:
- the proctor pauses a test
- the test taker click on the next button
- the current item state must be stored, so when the test is resumed the test taker retrieve its choices.

With this change, the session is checked and as it is suspended, the state is not stored.